src/api/export.ts

```diff
@@ -186,7 +187,7 @@ async function _export({
 					const cleaned = clean_html(body);
 
 					const base_match = /<base ([\s\S]+?)>/m.exec(cleaned);
-					const base_href = base_match && get_href(base_match[1]);
+					const base_href =  base_match ? get_href(base_match[1]) : '/'
```

Comment:
nit: don't need the extra space before `base_match` and should add a semi-colon to the end of the line